jcl/jpp_configuration.xml

```diff
@@ -342,6 +342,8 @@
 		<source path="src/openj9.dataaccess/share/classes"/>
 		<source path="src/openj9.sharedclasses/share/classes"/>
 		<source path="src/openj9.zosconditionhandling/share/classes"/>
+		<source path="src/openj9.cuda/share/classes"/>
+		<source path="src/openj9.gpu/share/classes"/>
```

Comment:
I don't think these should be added to any config besides this one (SIDECAR19-SE_RAWPLUSJ9).